contrib/cirrus/runner.sh

```diff
@@ -373,6 +373,19 @@ function logformatter() {
 # Handle local|remote integration|system testing in a uniform way
 dotest() {
     local testsuite="$1"
+    if [[ "$OS_RELEASE_ID" == "debian" ]]; then
+	# For debian envs pre-configure storage driver as overlay.
+	# See: Discussion here https://github.com/containers/podman/pull/18510#discussion_r1189812306
+	# for more details.
+	if [[ "$PRIV_NAME" == "rootless" ]] && [[ "$UID" -eq 0 ]]; then
```

Comment:
why eq 0? In this case $HOME would be incorrect.

Reply:
Removed it, its not needed.